src/main.rs

```diff
@@ -370,18 +364,6 @@ fn rocket() -> rocket::Rocket {
                 post_edit_override,
             ],
         )
-        .mount("/styles", StaticFiles::from("static/styles"))
-        .mount("/scripts", StaticFiles::from("static/scripts"))
-        .mount("/fonts", StaticFiles::from("static/fonts"))
-        .mount("/dict", StaticFiles::from("static/dict"))
-        .mount("/", StaticFiles::from("static/pwa").rank(20))
+        .mount("/", FileServer::from(relative!("static")).rank(20))
```

Comment:
The pwa directory needs to be mounted directly at `/`, and should not be available at `/pwa`

Reply:
I'm changing this to how you did it before, but it might be better to just move the pwa directory to a different parent directory, like `public`, and serve it from there.